LoopTestingKit/TestingScenarioInstance.swift

```diff
@@ -24,3 +66,13 @@ public struct TestingScenarioInstance {
         !pumpEvents.isEmpty
     }
 }
+
+public extension TestingScenarioInstance {
+    static var thirteenHourTrace: TestingScenarioInstance {
```

Comment:
Not used?

Reply:
Ok, just saw this is used in TidepoolOnboarding.  Since this is just being used for the marketing demo, it should probably be moved to TidepoolOnboarding, or TidepoolSupport

Reply:
That makes sense. I'll move it!

Reply:
Done